### PR TITLE
Added fan support, proper timeouts for commands

### PR DESCRIPTION
**Pull request by mrukavishnikov**
According to NooLite developers commands are sent on about 50ms (1 to send, 1 to receive response, and the same for repeating command)

### CODE REVIEW

File: NooLite_F/MTRF64/MTRF64Controller.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from NooLite_F import NooLiteFController, BrightnessDirection, ModuleMode, RemoteControllerListener, BatteryState
+from NooLite_F import NooLiteFController, BrightnessDirection, BrightnessDirection, ModuleMode, RemoteControllerListener, BatteryState
```
Comment (SergejPr) [inline NooLite_F/MTRF64/MTRF64Controller.py:1]: There is a duplication of BrightnessDirection.

File: NooLite_F/NooLiteFController.py [hunk 2]
```diff
@@ -314,7 +401,22 @@ def set_brightness(self, brightness: float, module_id: int = None, channel: int
...
+        :param broadcast: broadcast mode. If True then command will be send simultaneously to all modules that are binded with selected channel (default - False). If module_id is specified or mode is NOOLITE then broadcast parameter will be ignored.
+        :param brightness: speed level. The range of value is 0 .. 1.0
+        :param module_mode: module work mode, used to determine adapter mode for send command (default - NOOLITE_F).
+        :return: for nooLite-F command returns array which contains command result and module info for each module that are binded with selected channel. For nooLite modules returns nothing.
+        """
+        pass
+
+    @abstractmethod
+    def roll_rgb_color(self, module_id: int = None, channel: int = None, broadcast: bool = False,
+                       module_mode: ModuleMode = ModuleMode.NOOLITE_F) -> List[ResponseBaseInfo]:
```
Comment (SergejPr) [inline NooLite_F/NooLiteFController.py:419]: All these functions and their implementation are the duplication of the brightness function. The main idea of this class is hi level interface for all API commands. Api isn't provided speed commands, only brightness. So I don't see any sence in these methods. You can use brightness functions directly in your fan module the same way as you use dimmer configuration functions.

**Review by SergejPr — CHANGES_REQUESTED** [review]
Hi, thanks for adding new functionality. However I have left some comments. Please see them.

File: NooLite_F/NooLiteFController.py (hunk 2 above)
Comment (mrukavishnikov) [reply NooLite_F/NooLiteFController.py:419]: Sure, but HA will think about it as a light, not a fan.

Comment (SergejPr) [reply NooLite_F/NooLiteFController.py:419]: No, you will left your HA Fan module without changes, exclude call of NooLite module methods. For example, insted of  
```
responses = DEVICE.set_speed(int_speed / 255, None, self._config.get(CONF_CHANNEL),
                                     self._config.get(CONF_BROADCAST), _module_mode(self._config))
``` 
you will call 
```
responses = DEVICE.set_brightness(brightness / 255, None, self._config.get(CONF_CHANNEL),
                                          self._config.get(CONF_BROADCAST), _module_mode(self._config))
``` 
As result HA will thinks that works with fan, but will use brightness method for setting output voltage level.

Comment (SergejPr) [reply NooLite_F/NooLiteFController.py:419]: If you don't mind I can merge your changes and update it according to my point of view.

**Pull request merged**